ros2service/ros2service/verb/info.py

```diff
@@ -0,0 +1,61 @@
+# Copyright 2022 PickNik Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from ros2cli.node.strategy import NodeStrategy
+from ros2service.api import get_service_clients_and_servers
+from ros2service.api import ServiceNameCompleter
+from ros2service.verb import VerbExtension
+
+
+class InfoVerb(VerbExtension):
+    """Print information about a service."""
+
+    def add_arguments(self, parser, cli_name):
+        arg = parser.add_argument(
+            'service_name',
+            help="Name of the ROS service to print info about (e.g. '/talker/list_parameters')")
+        arg.completer = ServiceNameCompleter(
+            include_hidden_services_key='include_hidden_services')
+        parser.add_argument(
+            '-t', '--show-types', action='store_true',
+            help='Additionally show the service type')
+        parser.add_argument(
+            '-c', '--count', action='store_true',
+            help='Only display the number of service clients and service servers')
```

Comment:
Since this is `info` command, these should be always printed?

Reply:
`ros2 topic info` command always prints type and count, there is not even `opt-out` option.

Reply:
if possible, It seems good to be consistent in the same format and option. (https://docs.ros.org/en/rolling/Tutorials/Beginner-CLI-Tools/Understanding-ROS2-Topics/Understanding-ROS2-Topics.html#ros2-topic-info)